### DISCUSSION TITLE
Does Radix pass the accessibility compliance?

### DISCUSSION THREAD

**Discussion by bohdanbirdie** [radix-ui/primitives#2065] [Help]
Hey!
Thanks for making Radix open-source.

We been experimenting with it for a while, but the big question for us is the accessibility compliance. 
I understand it is possible to override the HTML tag and attributes, but is the defaults selected within the compliance?

For example Toast uses the `<ol>` container and `<li>` children. And the storybook accessibility testing tool already complains that the role is not a right one for the `<ol>` and etc.

So, I'm really curious about the compliance

**Comment by badsyntax** [comment]
I'm seeing the same accessiblity violation on Toast. Here's an axe report:

<img width="795" alt="Screenshot 2023-05-24 at 09 18 43" src="https://github.com/radix-ui/primitives/assets/102141/50bca4c1-441d-484b-83c8-b52100670b49">
